.travis.yml

```diff
@@ -1,14 +1,17 @@
 sudo: required
-dist: trusty
+dist: xenial
```

Comment:
It will use the xenial dist once Travis provides a container, which will probably happen in the next month, so at the moment it shouldn't matter.